site/styles/_meta-detail.scss

```diff
@@ -326,15 +326,17 @@
     }
     a {
       color: $body-color; 
-      font-weight: $font-weight-normal;
+      font-weight: bold;
+      text-decoration: underline;
+      background-color: transparent;
       text-decoration: underline;
       animation: fadein 2s;
       font-size: 12px;
       -moz-animation: fadein 2s; /* Firefox */
       -webkit-animation: fadein 2s; /* Safari and Chrome */
       -o-animation: fadein 2s; /* Opera */
       @include hover-focus {
-        color: rgba($body-color, 0.8); 
+        color: $secondary; 
```

Comment:
This should be `$body-color`... the client's secondary colour could easily be worse than the primary colour!